### ISSUE TITLE
feat: ion-datetime wheel appearance 

### ISSUE THREAD

**Issue by mikeangeloo** [ionic-team/ionic-framework#25945]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already include this feature request, without success.

### Describe the Feature Request

Hello team currently I had been working with the ion-datetime and have a lot of amazing things. It will be very useful I we can modify the styling of the ion-picker-internal, because is not meet the default font defined at global styles, we can change the Theme colors and background but no the size of the font, font-weight, font-size, font-family.

### Describe the Use Case

This feature will help to customize and meet the UI look & feel of each individual personalization.

### Related Code

`<ion-datetime
                        presentation="date"
                        [preferWheel]="true"
                    >
                    </ion-datetime>`

on sytles.scss
ion-datetime {
    color: #{$navy};
    font-weight: 700;
    --background: white;
    --ion-color-base: #{$navy} !important;
    --ion-color-step-150: #{$light-teal};
}

The result:
![image](https://user-images.githubusercontent.com/13010699/190266099-615b3e35-6ad1-46f8-9bc1-0dfd0639fe14.png)

The expected result:
![image](https://user-images.githubusercontent.com/13010699/190267446-5320f431-5f35-46e5-a796-fcb3a385d190.png)

### Additional Information

This element should be customizable or have ::part element to get into it

- picker-item
- picker-item-active
- <div class="picker-before"></div>
- <div class="picker-after"></div>
- <div class="picker-highlight"></div>

![image](https://user-images.githubusercontent.com/13010699/190267740-f2ef66ff-c5d9-4a00-9725-78db187616c3.png)

**Comment by sjdrew** [comment]
I agree with this also. In fact the picker highlight in MD mode is never used and we our finding users do not know which row is the highlight as not enough contrast difference. 
<img width="268" alt="image" src="https://user-images.githubusercontent.com/20103874/194602452-9ff19203-638b-4f38-aa18-1b881ed77964.png">

In this example many users think the black text is the selected not the green text (primary color).  IOS version is fine as you highlight the row.

Also mode="ios" does not work either to force ios mode on the ion-datetime (has no effect on the popover).

**Comment by thalisvilela** [comment]
+1. In testing with some users, they found difficult to know which row is selected.

**Comment by AssafGolani** [comment]
+1, in my case, the picker-before inside shadow-root has for some reason a white background with little opacity, this makes the visibility for presentation 'time' impossible to read, and I don't understand why this happens.

`:host .picker-before {
    background: linear-gradient(to bottom, var(--background, var(--ion-background-color, #fff)) 20%, rgba(var(--background-rgb, var(--ion-background-color-rgb, 255, 255, 255)), 0.8) 100%);
}`

**Comment by ogaguinaga** [comment]
Are there any news on this issue? how do you change the ion-datetime component wheel styles?

**Comment by loyaj** [comment]
I'm in the same boat. Since ion-picker is a scoped component, I've been able to adjust the styles for standalone pickers. But I also need to be able to style the pickers that are part of ion-datetime, which is a shadow component.

**Comment by amandaejohnston** [comment]
Features to style the wheel pickers have been added via https://github.com/ionic-team/ionic-framework/pull/27529 and will be available in an upcoming minor release of Ionic. Thank you!
